tests/x11/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -93,6 +93,7 @@ sub run {
     # launch components from Activities overview
     $self->open_overview();
     type_string "base";
+    wait_still_screen 10;
```

Comment:
This wastes at least 10s. Keep in mind that this is no simple sleep but a wait for a "still screen"